mitmproxy/contentviews/base.py

```diff
@@ -49,18 +49,21 @@ def format_dict(
     ]
     entries, where key is padded to a uniform width.
     """
-    max_key_len = max(len(k) for k in d.keys())
-    max_key_len = min(max_key_len, KEY_MAX)
-    for key, value in d.items():
-        if isinstance(key, bytes):
-            key += b":"
-        else:
-            key += ":"
-        key = key.ljust(max_key_len + 2)
-        yield [
-            ("header", key),
-            ("text", value)
-        ]
+    try:
+        max_key_len = max(len(k) for k in d.keys())
+        max_key_len = min(max_key_len, KEY_MAX)
+        for key, value in d.items():
+            if isinstance(key, bytes):
+                key += b":"
+            else:
+                key += ":"
+            key = key.ljust(max_key_len + 2)
+            yield [
+                ("header", key),
+                ("text", value)
+            ]
+    except ValueError:
+        print("Body is blank")  # FIXME: Better message
```

Comment:
I didn't look into this issue, but in general situation you should use:
`ctx.log.error` (`info`, `alert`, `debug` or `warn`) or `raise exceptions.CommandError("Error message")`

Reply:
Importing ctx is throwing me an error `AttributeError: module 'mitmproxy.options' has no attribute 'Options'` . Once I know the root cause of it, I will change